tests/e2e/utils/set-search-console-property.js

```diff
@@ -0,0 +1,14 @@
+
+import { wpApiFetch } from './wp-api-fetch';
```

Comment:
Same here: Empty line before and no `Internal dependencies` docblock (mandatory anyway once #217 lands).